enterprise/frontend/src/embedding-sdk/cli/steps/setup-license.ts

```diff
@@ -0,0 +1,68 @@
+import { input } from "@inquirer/prompts";
+import chalk from "chalk";
+import toggle from "inquirer-toggle";
+import open from "open";
+
+import type { CliStepMethod } from "embedding-sdk/cli/types/cli";
+
+import { printEmptyLines, printInfo } from "../utils/print";
+import { propagateErrorResponse } from "../utils/propagate-error-response";
+
+const trialUrl = `https://store.metabase.com/checkout?plan=pro&deployment=self-hosted`;
+const trialUrlWithUtm = `${trialUrl}&utm_source=product&utm_medium=checkout&utm_campaign=embedding-sdk&utm_content=embedding-sdk-cli`;
```

Comment:
Do not print the UTM tags in the console as it's clutter, we open the URL with UTM tags in the user's browser of choice instead.